pandas/util/_test_decorators.py

```diff
@@ -99,36 +99,40 @@ def _skip_if_no_scipy():
 
 def skip_if_no(package, min_version=None):
```

Comment:
can you add annotations (return value in particular)

Reply:
> can you add annotations (return value in particular)

done